internal/locate/region_request3_test.go

```diff
@@ -1018,3 +1019,58 @@ func (s *testRegionRequestToThreeStoresSuite) TestAccessFollowerAfter1TiKVDown()
 		s.Equal(0, bo.GetTotalBackoffTimes())
 	}
 }
+
+func (s *testRegionRequestToThreeStoresSuite) TestStaleReadFallback() {
```

Comment:
This test was also passed before this PR.

Reply:
It would fail after removing the `!state.option.leaderOnly` check which is the early version.

Reply:
ok